changelog/changes/2023-05-12-coreos-cloudinit.md

```diff
@@ -0,0 +1 @@
+- Update coreos-cloudinit to the [latest commit](https://github.com/flatcar/coreos-cloudinit/commit/89319292b9bca85a7a1f5f8a47c459dd45a8cc7a). This commit changes how coreos-cloudinit sets the hostname fetched from the metadata service. The short form hostname is now set as opposed to the FQDN.
```

Comment:
It's a bit too verbose, the details on how the change is brought to the user is not really relevant to them.
```suggestion
- Changed coreos-cloudinit to now set the short hostname instead of the FQDN when fetched from the metadata service ([coreos-cloudinit#19](https://github.com/flatcar/coreos-cloudinit/pull/19))
```